packages/google_sign_in/google_sign_in_ios/ios/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -116,18 +112,14 @@ - (void)initializeSignInWithParameters:(nonnull FSIInitParams *)params
   if (configuration != nil) {
     self.requestedScopes = [NSSet setWithArray:params.scopes];
     self.configuration = configuration;
-  } else {
-    *error = [FlutterError errorWithCode:@"missing-config"
-                                 message:@"GoogleService-Info.plist file not found and clientId "
-                                         @"was not provided programmatically."
-                                 details:nil];
```

Comment:
The Google Sign In SDK now gets the configuration settings within the SDK itself from the Info.plist and that's now the preferred way to do it.

Reply:
FYI I moved `GoogleService-Info.plist` from the Runner target to the RunnerTests target, so we can still use it for the example tests.